network-api/networkapi/wagtailpages/README.md

```diff
@@ -86,7 +86,7 @@ In addition to the standard Wagtail-supplied blocks, the `customblocks.py` files
 
 This block sets up a button that links out to a URL, using `wagtailpages/templates/wagtailpages/blocks/link_button_block.html` as its rendering template.
 
-#### 2. ImageTextBlock
+#### 2. ImageTextBlock2
```

Comment:
I think we can leave this as ImageTextBlock since that'll be the name of our updated ImageText2 streamfield component [after we rename it](https://github.com/mozilla/foundation.mozilla.org/issues/2693#issue-412555535). What do you think?

Reply:
might as well rename it back in that PR.